.github/workflows/golangci-lint.yml

```diff
@@ -19,4 +19,4 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@3a919529898de77ec3da873e3063ca4b10e7f5cc #pin@3.7.0
         with:
-          version: v1.55
+          version: v1.56
```

Comment:
Earlier in this file there is a line that reads

```
go-version: "1.21"
```

This needs to be updated to 1.22.

Reply:
Silly mistake sorry.